trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/get-is-checkbox-checked.ts

```diff
@@ -0,0 +1,5 @@
+export const getIsCheckboxChecked = (checkedFeeRecordIds: Set<number>): ((feeRecordIds: number[]) => boolean) => {
```

Comment:
Can we add some documentation, 

Also should this be called something like "every checkbox is checked"?